From 44eb1166ee002a4cfd1e7614eca6d64d97c38530 Mon Sep 17 00:00:00 2001
From: Michael Scott <mike@foundries.io>
Date: Wed, 10 May 2023 17:14:00 -0700
Subject: [PATCH 2/6] imx8mp_var_dart: update dfu_alt_info for split bootloader
 updates

Update dfu_alt_info to allow updating SPL/imx-boot and u-boot.itb via
DFU (which is also used by EFI capsule updates).

Signed-off-by: Michael Scott <mike@foundries.io>
---
 include/configs/imx8mp_var_dart.h | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/include/configs/imx8mp_var_dart.h b/include/configs/imx8mp_var_dart.h
index 63be7f6265d..84f0a7b9916 100644
--- a/include/configs/imx8mp_var_dart.h
+++ b/include/configs/imx8mp_var_dart.h
@@ -110,7 +110,8 @@
 	"m7_addr=0x7e0000\0" \
 	"m7_bin=hello_world.bin\0" \
 	"use_m7=no\0" \
-	"dfu_alt_info=mmc 2=1 raw 0x40 0x1000 mmcpart\0" \
+	"dfu_alt_info=mmc 2=imx-boot raw 0x0 0x300 mmcpart 1;" \
+		"u-boot-itb raw 0x300 0xD00 mmcpart 1\0" \
 	"loadm7bin=" \
 	         "load mmc ${mmcdev}:${mmcpart} ${loadaddr} ${bootdir}/${m7_bin} && " \
 	         "cp.b ${loadaddr} ${m7_addr} ${filesize}; " \
-- 
2.40.1

